src/components/organisms/news/news.scss

```
@@ -0,0 +1,60 @@
+@use "../../variables" as *;
+.news {
+    background-color: $white_smoke;
+    .news-and-events-title {
+        h2 {
+            font-size: $fontsize-xxl;
+            .caption {
+                font-size: $fontsize-m;
+            }
+            @media (max-width: $breakpoint-md) {
+                font-size: $fontsize-l;
+                .caption {
+                    font-size: $fontsize-m;
+                }
+            }
+        }
+    }
+    .content {
+        background-color: $white;
+        margin-bottom: 30px;
+        .news-link {
+            text-decoration: none;
```

```suggestion
        .news-link {
            text-decoration: none;
                &:hover {
                    .info {
                        color: $orange_bistre;
                    }
              }
```
こうすると、クリックできる範囲とホバーで文字色が変わる範囲を一致させることができます。